File: app/src/test/java/com/example/myapplication/core/feedback/WorkoutFeedbackTest.kt
package com.example.myapplication.core.feedback

import org.junit.Assert.assertEquals
import org.junit.Test

class WorkoutFeedbackTest {
    @Test
    fun `rating score preserves easy right hard ordering`() {
        assertEquals(-1, WorkoutDifficulty.EASY.score)
        assertEquals(0, WorkoutDifficulty.RIGHT.score)
        assertEquals(1, WorkoutDifficulty.HARD.score)
    }
}

File: flutter/test/core/feedback/workout_feedback_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:gym_app/core/model/feedback_models.dart';

void main() {
  test('rating score preserves easy right hard ordering', () {
    expect(WorkoutDifficulty.easy.score, -1);
    expect(WorkoutDifficulty.right.score, 0);
    expect(WorkoutDifficulty.hard.score, 1);
  });
}
